docs/dgx-pod.md

```diff
@@ -77,129 +79,130 @@ The DeepOps service container "DGXie" provides DHCP, DNS, and PXE services to th
+
    ./scripts/setup.sh
+
    ```
 
-3. Copy and version control the configuration files. The `config/` directory is ignored by the deepops git repo. Create a seperate git repo to track local configuration changes.
 
-   ```sh
-   cp -r config.example/ config/
-   cd config/
-   git init .
-   git add .
-   git commit -am 'initial commit' && cd ..
-   ```
+3. Edit the Ansible inventory file.
+
+Ansible uses an inventory which outlines the servers in your cluster. The setup script from the previous step will copy an example inventory configuration to the `config` directory. 
+
+The cluster should ideally use DNS, but you can also explicitly set server IP addresses in the inventory file.
 
-4. Modify the `config/inventory` file to set the cluster server hostnames, and optional per-host info like IP addresses and network interfaces. The cluster should ideally use DNS, but you can also explicitly set server IP addresses in the inventory file.
+```sh
+# Edit inventory and add nodes to the "ALL NODES and "KUBERNETES" sections
+   # Note: Etcd requires an odd number of servers (https://kubernetes.io/docs/tasks/administer-cluster/configure-upgrade-etcd/#prerequisites)
+
+   vi config/inventory
+   
+   # (optional) Modify `config/group_vars/*.yml` to set configuration parameters.  The cluster-wide global config resides in the `all.yml`
 
-   Optional inventory settings:
+```
 
-   * Use the `ansible_host` variable to set alternate IP addresses for servers or for servers which do not have resolvable hostnames
+     Optional inventory settings:
+
+   * Use the `ansible_host` variable to set alternate IP addresses for servers or for servers which do not have resolvable hostnames (i.e. 'mgmt01     ansible_host=10.0.0.1')
    * Use the `ib_bond_addr` variable to configure the infiniband network adapters with IPoIB in a single bonded interface
 
-5. Configure cluster parameters by modifying the various yaml files in the `config/group_vars` directory. The cluster-wide global config resides in the `all.yml` file, while group-specific options reside in the other files. File names correspond to groups in the inventory file, i.e. `[dgx-servers]` in the inventory file corresponds with `config/group_vars/dgx-servers.yml`.
 
 ### 2. Prepare the management node(s)
 
 The configuration assumes a single cpu-only management server, but multiple management servers can be used for high-availability. Ensure that the inventory file is edited accordingly if using more than one management server.
 
-Install the latest version of Ubuntu Server 18.04 LTS on each management server. Be sure to enable SSH and record the user and password used during install.
+Install the latest version of Ubuntu Server 18.04 LTS on each management server. Be sure to enable SSH `apt install openssh-server` and record the user and password used during install.
```

Comment:
Can you put the `apt install` in parentheses?

Reply:
do you mean () or ""  as it is it's wrapped in  ` ` to hight light as a code segment.

Reply:
()